Pod/Classes/Deselection/UIViewController+Deselection.swift

```diff
@@ -30,64 +30,58 @@
 
 import UIKit
 
+public protocol SmoothlyDeselectableItems {
+    var indexPathsForSelectedItems: [IndexPath]? { get }
```

Comment:
Also i think there are other use cases for this (maps for example) so I think it's valid to keep it open